index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+<body>
+  <h1>THIS IS MY DOPE SITE</h1>
+  <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Facilis qui non expedita pariatur similique omnis nobis, iste, consectetur quisquam repellendus corporis error nihil aut accusamus quia placeat. Minima, possimus cum?Lorem ipsum dolor sit amet, consectetur adipisicing elit. Illum reiciendis odio facere inventore, vel veniam expedita quaerat libero magni illo eveniet quod eum, et reprehenderit eligendi sint, officiis doloribus exercitationem?Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolores tempore vitae quod doloribus nihil a non. Quidem, atque eaque quam a voluptatibus ipsum iste hic rem culpa adipisci blanditiis iusto!</p>
```

Comment:
i think we can change this from a paragraph to an h5 @OSP123

Reply:
You're so right! Let me change this.